### ISSUE TITLE
`bowtie summary` should output the summary along with version numbers of implementations.

### ISSUE THREAD

**Issue by adwait-godbole** [bowtie-json-schema/bowtie#1267]
We don't want to do it always, just if the same implementation appears multiple times with different versions like shown below...

```
⊙  bowtie suite -i ruby-json_schemer -i image:ruby-json_schemer:0.2.25 -i image:ruby-json_schemer:0.2.17 7 | bowtie summary --show failures
Ran 252 test cases.
                      Bowtie                       
┏━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┓
┃ Implementation      ┃ Skips ┃ Errors ┃ Failures ┃
┡━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━╇━━━━━━━━╇━━━━━━━━━━┩
│ json_schemer (ruby) │ 0     │ 0      │ 0        │
│ json_schemer (ruby) │ 0     │ 23     │ 2        │
│ json_schemer (ruby) │ 0     │ 24     │ 3        │
└─────────────────────┴───────┴────────┴──────────┘
```

Ref: [#1250 (comment)](https://github.com/bowtie-json-schema/bowtie/pull/1250#issuecomment-2154358694)

**Comment by Julian** [comment]
It's always possible -- though of course such a thing would be a bug (in our harness) and we'd fix it. But that'd be exactly the kind of thing that would cause us to re-release the *old* image and want to recalculate the results.

**Comment by adwait-godbole** [comment]
Fixed in #1284
